src/index.html

```diff
@@ -17,16 +17,20 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form>
+      <input
+        type="text"
+        class="search-bar search-bar--big"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        type="text"
+        class="search-bar search-bar--small"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
```

Comment:
Both input fields have the same placeholder text. If this is not intentional, consider providing different placeholder texts to better guide the user on what each input field is for.